.drone.yml

```diff
@@ -8,6 +8,15 @@ pipeline:
     when:
       event: [push, pull_request]
 
+  build_docker_compose_acceptance test:
+    image: docker/compose
+    commands:
+      - docker-compose up -d --build chrome-browser app redis
```

Comment:
back in the day, I remember people having issues running docker-compose in ACP. Worth checking if that's still possible. I don't see why not to be fair